bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -342,75 +307,30 @@ spec:
             {{- end }}
           {{- end }}
           ports:
-            - containerPort: 27017
-              name: mongodb
+            - name: mongodb
+              containerPort: {{ .Values.containerPorts.mongodb | int }}
```

Comment:
In the standalone you added support for user custom containerPorts. If you'd like to continue with that approach, I'd suggest adding `extraContainerPorts` here too.

Reply:
My bad, that code shouldn't be pushed. I didn't revert it in the standalone version. Sorry about that